packages/apps/docs/package.json

```diff
@@ -56,7 +54,8 @@
     "remark-frontmatter": "~4.0.1",
     "remark-gfm": "~3.0.1",
     "shiki": "~0.14.2",
-    "styled-components": "~5.3.10"
+    "styled-components": "~5.3.10",
```

Comment:
what are we using styled components for?

Reply:
it is used for the component that creates the API Swagger page.

Reply:
It's not referenced anywhere, so looks like it shouldn't be in this list? Not part of this PR, though.